flight/tests/coordinate_conversions/unittest.cpp

```diff
@@ -85,3 +85,306 @@ TEST_F(RneFromLLATest, Equator) {
   ASSERT_NEAR(0, Rne[2][1], eps);
   ASSERT_NEAR(0, Rne[2][2], eps);
 };
+
+
+//// Test fixture for LLA lineraization()
+class LLALineraization : public CoordConversion {
+protected:
+  virtual void SetUp() {
+  }
+
+  virtual void TearDown() {
+  }
+};
+
+TEST_F(LLALineraization, Equator_float) {
+  // Test location is Equator
+  float homeLLA_D[] = {0, 0, 0};
+  float dN = 0.0111319490793274;
+  float dE = 0.0111319490793274;
+  float dD = -1;
+
+  float T[3];
+  LLA_linearization_float(homeLLA_D[0], homeLLA_D[2], T);
+
+  float eps = 0.000001f;
+
```

Comment:
Probably should add a comment that you're looking for the ratio between expected and returned values, and not absolute differences.